13-tool-usage-cycle-tracking/README.md

```diff
@@ -377,28 +456,29 @@ This is an alert event (due to the presence of the `alert` property), which will
 
 ```json
 { "req": "note.add", "file":"power.qo", "sync": true, "body": {
-  "current":2.34, "frequency":60, "power":56.67, "voltage":230.12, "alert":"overcurrent,power", "instance":2, "active":true
+  "current":2.34, "frequency":60, "power":56.67, "voltage":230.12, "alert":"overcurrent,power", "instance":2, "active":true, "vibration":"normal", "vibration_raw":123
 }}
 ```
 
 This will send a SMS that looks like this:
 
-> Power alert from machine-12 equipment powered: yes: overcurrent,power. 120V, 25A, 3000W.
+> Power alert from <serial-number> tool active: yes: overcurrent,power. 120V, 25A, 3000W. vib.: normal,123.
 
 These are the parts of the message:
 
-* The first part of the message indicates which device the alert pertains to by its serial number, here "machine-12".
-
-* The next part indicates which monitor instance generated the alert, here it is "equipment", with "utility supply" as the other option. These names relate to the instance numbers, and here we are using 1 and 2 for utility supply and equipment respectively. If your monitors are configured with different I2C addresses, you will need to edit the names "instance-3" and "instance-4" in the jsonata script.
+* The first part of the message indicates which device the alert pertains to by its serial number, here <serial-number>.
```

Comment:
Same problem here. I think putting `<serial-number>` in code tags will fix this one though.

Reply:
I used `&lt;` etc.